src/main/java/org/mafagafogigante/dungeon/schema/rules/SpecificIdJsonRule.java

```diff
@@ -0,0 +1,31 @@
+package org.mafagafogigante.dungeon.schema.rules;
+
+import org.mafagafogigante.dungeon.game.Id;
+
+import com.eclipsesource.json.JsonValue;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+class SpecificIdJsonRule extends IdJsonRule {
+
+  private final Set<Id> ids = new HashSet<>();
+
+  SpecificIdJsonRule(Collection<String> stringIds) {
+    for (String stringId : stringIds) {
+      ids.add(new Id(stringId));
+    }
+  }
+
+  @Override
+  public void validate(JsonValue value) {
+    super.validate(value);
+    Id testId = new Id(value.asString());
+    boolean isValueExist = ids.contains(testId);
```

Comment:
`exists` would be a more grammatically correct name. I don't have any problem understanding you, but the codebase should stay as readable as possible.